src/main/java/net/dv8tion/jda/internal/interactions/CommandInteractionImpl.java

```diff
@@ -73,6 +74,7 @@ public CommandInteractionImpl(JDAImpl jda, DataObject data)
 
         parseResolved(jda, resolveJson);
         parseOptions(options);
+        buildText();
```

Comment:
This should not be a field, it should be done in the getter just like getCommandPath. Most users have no use for this string so it makes no sense to generate it for every command interaction.

Reply:
Removed